### PR TITLE
General polish

### PR DESCRIPTION
**Pull request by DavidVorick**
Nothing too major in this pr, just cleaning up a few things.

I disabled transactions because they weren't working, there were a decent amount of errors, and I couldn't figure out how to get it working.

### CODE REVIEW

File: app/js/pluginManager.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ function PluginManager() {
 
 		// Display any console logs from the plugin
 		plugin.on('console-message', function(event) {
-			console.log(plugin.name + ' plugin logged> ', event.message);
+			console.log(plugin.name + ' plugin logged> ' + event.message);
```
Comment (Mingling94) [inline app/js/pluginManager.js:141]: Either way works

Comment (DavidVorick) [inline app/js/pluginManager.js:141]: nope.

using a comma meant that the event message was never displayed, interfered with development.

File: index.js [hunk 2]
```diff
@@ -44,6 +45,27 @@ function startMainWindow() {
...
+            { label: "Undo", accelerator: "Command+Z", selector: "undo:" },
+            { label: "Redo", accelerator: "Shift+Command+Z", selector: "redo:" },
+            { type: "separator" },
+            { label: "Cut", accelerator: "Command+X", selector: "cut:" },
+            { label: "Copy", accelerator: "Command+C", selector: "copy:" },
+            { label: "Paste", accelerator: "Command+V", selector: "paste:" },
+            { label: "Select All", accelerator: "Command+A", selector: "selectAll:" }
+        ]}
+    ];
+    Menu.setApplicationMenu(Menu.buildFromTemplate(template));
```
Comment (Mingling94) [inline index.js:68]: Oh Macs, you and your silly alternate ways of doing things

File: app/js/pluginManager.js (hunk 1 above)
Comment (Mingling94) [inline app/js/pluginManager.js:141]: Oh right! That never occurred to me that this caused it, Nice!

**Comment by Mingling94** [comment]
Looks good! I'll get the transactions working up to the new API PR later today

**Pull request merged**